src/cards/base/standardProjects/GreeneryStandardProject.ts

```diff
@@ -21,6 +21,14 @@ export class GreeneryStandardProject extends StandardProjectCard {
     });
   }
 
+  public canPayWith(player: Player) {
```

Comment:
Not sure why I am just now recommending this but we should use the `override` keyword introduced in typescript 4.3 anywhere we can. There is a flag we can turn on to require it's usage. Should help with reading and help us find places we override.
```suggestion
  public override canPayWith(player: Player) {
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-3.html

Reply:
Yeah let's.